infrastructure/ansible/roles/traffic_ops/templates/cdn.conf.j2

```diff
@@ -45,6 +46,12 @@
       "email_from" : "{{ to_email_from }}",
       "no_account_found_msg" : "{{ to_account_missing_msg }}"
    },
+   "smtp" : {
+      "enabled" : {{ to_smtp_enabled | lower }},
```

Comment:
The `to_smtp_enabled` variable should be treated as a boolean (and enforced to run through a boolean filter) before running through `lower` to get `true || false`